src/containers/Accounts/AMM/AMMAccounts/index.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { useParams, useRouteMatch } from 'react-router'
+import { useTranslation } from 'react-i18next'
+import '../../styles.scss'
+import formatBalance from '../../../../rippled/lib/txSummary/formatAmount'
+import {
+  getAccountTransactions,
+  getAMMInfo,
+} from '../../../../rippled/lib/rippled'
+import { Tabs } from '../../../shared/components/Tabs'
+import { analytics, ANALYTIC_TYPES, formatAsset } from '../../../shared/utils'
+import SocketContext from '../../../shared/SocketContext'
+import { ERROR_MESSAGES } from '../../Errors'
+import NoMatch from '../../../NoMatch'
+import {
+  AMMAccountHeader,
+  AmmDataType,
+} from './AMMAccountHeader/AMMAccountHeader'
+import { AccountTransactionTable } from '../../AccountTransactionTable'
+
+const getErrorMessage = (error: string) =>
+  ERROR_MESSAGES[error] || ERROR_MESSAGES.default
+
+export const AMMAccounts = (props: any) => {
+  const { id: accountId, tab = 'transactions' } = useParams<{
+    id: string
+    tab: string
+  }>()
+  const { path = '/' } = useRouteMatch()
+  const { t } = useTranslation()
+  const mainPath = `${path.split('/:')[0]}/${accountId}`
+  const rippledSocket = useContext(SocketContext)
+  const [data, setData] = useState<AmmDataType>()
+  const [error, setError] = useState<any>()
+  const { language } = props
+
+  useEffect(() => {
+    let asset1: { currency: string; issuer?: string }
+    let asset2: { currency: string; issuer?: string }
+    let ammData: any
+
+    /*
+    Get the first account transaction which in this case should be AMMCreate. From this we get
+    the two assets in the asset pool.
+    */
+    getAccountTransactions(rippledSocket, accountId, 1, undefined, true)
```

Comment:
I'm not convinced that this will actually get you the first transaction for the account when there are multiple pages of transactions.

From the docs on the `forward` param:
```
If set to true, returns values indexed with the oldest ledger first. Otherwise, the results are indexed with the newest ledger first. (Each page of results may not be internally ordered, but the pages are overall ordered.)
```

Reply:
Also, this should really be added to `amm_info` - a way to search by account ID or object ID.

Reply:
Great point on the full history

Reply:
yea i agree on the second point. it's actually something I brought up at the time but it didn't make sense to increase the ledger size for this specific use case. 

Good find on the forward flag - It seems like it's still guaranteed to be in the first page. I'll add a search function that iterates through all the transactions within the page results and finds AMMCreate.

Reply:
This method though will still be an issue if the node is not full history.  While this is not an issue for the fully supported devnets this will be an issue for custom and sidechains.

Reply:
done

Reply:
Does it need to increase the ledger size if it's just an alternate form of request for `amm_info`?

Reply:
You'd have to talk to Greg I can't comment on that.